_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -12010,13 +12019,12 @@
 /obj/structure/rack,
 /obj/item/clothing/mask/surgical,
 /obj/item/clothing/suit/apron/chef,
-/turf/open/floor/plating,
+/turf/open/floor/iron/showroomfloor,
 /area/maintenance/starboard/secondary)
 "cmb" = (
-/obj/machinery/chem_master{
-	pixel_x = -4
-	},
-/turf/open/floor/plating,
+/obj/structure/frame/machine,
+/obj/item/circuitboard/machine/chem_dispenser,
```

Comment:
```suggestion
/obj/item/circuitboard/machine/chem_master,
```
Did you mean to put a chem-master board in here(as to allow separation of chemicals)? A whole chem-dispenser makes the setup rival the pharmacy, and removes most of the ghetto in ghetto-chem.

Reply:
Thanks for the catch, I'll make this change right away.